calico/datadog_checks/calico/check.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+class CalicoCheck(OpenMetricsBaseCheckV2):
+    def __init__(self, name, init_config, instances=None):
+        METRICS_MAP = {'felix_active_local_endpoints': 'felix_active_local_endpoints'}
+
+        super(CalicoCheck, self).__init__(
+            name,
+            init_config,
+            instances,
+            default_instances={
+                'calico-felix': {
+                    'prometheus_url': 'http://localhost:9091/metrics',
+                    'namespace': 'kube-system',
+                    'metrics': [METRICS_MAP],
+                }
+            },
+            default_namespace='kube-system',
```

Comment:
```suggestion
            default_namespace='calico',
```

Reply:
I removed locally this `default_namespace` parameter since `ddev test calico` refuse it, I think it must have been removed in OpenMetricsBaseCheckV2